### PR TITLE
add check minimum 8gb sd storage

### PR DESCRIPTION
**Pull request by mimefly**

### CODE REVIEW

**Comment by mimefly** [comment]
Test comment

File: scripts/burn-image.sh [hunk 1]
```diff
@@ -94,6 +94,16 @@ if [ "$disk_choice" == "" ] || [ "${sd_disks[disk_choice]}" == "" ]; then
 fi
 
 chosen_disk=${sd_disks[disk_choice]}
+
+checksize=$(echo "$size" | awk '{ print substr( $0, 1, length($0)-2 ) }')
+if [[ $checksize -lt 5500 ]]; then
```
Comment (markbaaijens) [inline scripts/burn-image.sh:99]: 5500 does not seem to resemble the 8gb mentioned in the error message?

File: scripts/burn-image.sh [hunk 2]
```diff
@@ -94,6 +94,16 @@ if [ "$disk_choice" == "" ] || [ "${sd_disks[disk_choice]}" == "" ]; then
...
+    echo "Your chosen disk $chosen_disk has a size of $size"
+    echo "Cannot continue. Please re-run this script after selecting an SD-card with at least 8gb capacity."
```
Comment (markbaaijens) [inline scripts/burn-image.sh:101]: It is not really 'selecting' another sd-card. Maybe 'using' is a better word. Or perhaps you have another suggestion?

**Comment by mimefly** [comment]
Deze comment is via de app toegevoegd

File: scripts/burn-image.sh (hunk 2 above)
Comment (markbaaijens) [inline scripts/burn-image.sh:101]: Nitpic.... it is 'a SD-card', not 'an SD-card'

File: scripts/burn-image.sh [hunk 3]
```diff
@@ -94,6 +94,16 @@ if [ "$disk_choice" == "" ] || [ "${sd_disks[disk_choice]}" == "" ]; then
 fi
 
 chosen_disk=${sd_disks[disk_choice]}
+
+checksize=$(echo "$size" | awk '{ print substr( $0, 1, length($0)-2 ) }')
```
Comment (markbaaijens) [inline scripts/burn-image.sh:98]: It is presumed that size is given in MB. But what if it is KB? Or GB?

**Review by markbaaijens — APPROVED** [review]

**Pull request merged**